forte/data/readers/audio_reader.py

```diff
@@ -30,6 +29,21 @@
 class AudioReader(PackReader):
     r""":class:`AudioReader` is designed to read in audio files."""
 
+    def initialize(self, resources, configs):
+        # pylint: disable=attribute-defined-outside-init
+        super().initialize(resources, configs)
+        try:
+            import soundfile  # pylint: disable=import-outside-toplevel
```

Comment:
I think we can install at the class scope (line 31), then we don't need to store `self._soundfile` to access it inside the class.

Reply:
Seems like it will trigger an error when building docs: https://github.com/asyml/forte/runs/4783761361?check_suite_focus=true#step:6:37